src/main/java/dev/j3fftw/litexpansion/ItemSetup.java

```diff
@@ -44,6 +45,7 @@ public void init() {
 
     private void registerTools() {
         new CargoConfigurator().register(LiteXpansion.getInstance());
+        new GlassCutter().register(LiteXpansion.getInstance());
```

Comment:
var this aswell

Reply:
What do you mean

Reply:
oh, LiteXpansion.getInstance()

Reply:
You will register more items soon so var .register(....